src/types/aws/ap/registration.ts

```diff
@@ -137,7 +137,7 @@ export type DoneFSAInquiry = Append<DoneOrgDetails, FSAInquiry, {}>;
 export type DoneDocs = Append<DoneFSAInquiry, TDocumentation, {}>;
 export type DoneBanking = Append<DoneDocs, BankingDetails, {}>;
 
-export type SubmissionDetails = { Email: string; EndowmentId: number };
+export type SubmissionDetails = { Email: string; EndowmentId?: number };
```

Comment:
probably best to add `Approved` state type after `InReview` to make things clearer but this would require more changes - could be in another PR